Legacy/util/src/main/java/org/bonej/geometry/FitEllipsoid.java

```diff
@@ -41,8 +41,10 @@ OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
  */
 public final class FitEllipsoid {
 
+	/** Random number generator*/
 	private static Random rng = new Random();
 
+	/** Constructor */
 	private FitEllipsoid() {}
```

Comment:
It's not necessary to comment `private` fields or methods. Especially if they're trivial.